### ISSUE TITLE
Accessibility - Goods List - When Searching TalkBack Does Not Announce Goods Found

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#82]
When searching for a country on the Goods screen TalkBack does not tell the user after they typed a letter that "X number of goods were found". Also, there is not an announcement of how many were found when using the "Search" button either.

Also, why are the Goods search finding products have a letter in the name instead of only at the start of the name like the country search does? If I'm typing in a search field and type a letter, I expect the word to start with that letter and not give me a list of products that start or have the letter in it.

**Comment by jfc3-dol** [comment]
TalkBack now announces the number of results found as you type letters, but not when you tap the "search" button, so I'm going to leave the issue open.

Also, why does the search find items that have the letters typed in them, and not only those that start with that letter? Most people would be typing letters for items that start with what they typed.

**Comment by jfc3-dol** [comment]
@pbhatt17 

When a user enters a search, it reads the number of items found, and when they use the Search button for the Goods screen or the Countries screen as expected.

So I'm going to close the issue.
